### PR TITLE
Add CPU test for swin_tiny_2 model

### PR DESCRIPTION
**Pull request by pdeviTT**

### CODE REVIEW

File: forge/test/models/pytorch/vision/swin/test_swin.py [hunk 1]
```diff
@@ -2,37 +2,84 @@
...
+    # STEP 3: Run inference on Tenstorrent device
     compiled_model = forge.compile(model, sample_inputs=inputs, module_name="pt_swin_tiny_patch4_window7_224")
+    verify(inputs, model, compiled_model)
+
+
+@pytest.mark.nightly
+@pytest.mark.model_analysis
+@pytest.mark.xfail(reason="AssertionError: Data mismatch on output 0 between framework and Forge codegen")
+@pytest.mark.parametrize("variant", ["microsoft/swinv2-tiny-patch4-window8-256"])
+def test_swin_v2_tiny_4_256_hf_pytorch(variant, test_device):
```
Comment (vkovinicTT) [inline forge/test/models/pytorch/vision/swin/test_swin.py:38]: Please just remove `test_device` from function params, as it isn't used anywhere. Remove it in other tests as well.

**Review by vkovinicTT — CHANGES_REQUESTED** [review]
Friendly reminder: 
- try not to load images in future, instead just create random tensor, as we don't want to have external images in our repo :)

File: forge/test/models/pytorch/vision/swin/test_swin.py [hunk 2]
```diff
@@ -2,37 +2,84 @@
...
+    inputs = load_image(url, feature_extractor)
 
-    # STEP 4: Run inference on Tenstorrent device
-    img_tensor = feature_extractor(images=image, return_tensors="pt").pixel_values
-    print(img_tensor.shape)
-
-    inputs = [img_tensor]
+    # STEP 3: Run inference on Tenstorrent device
     compiled_model = forge.compile(model, sample_inputs=inputs, module_name="pt_swin_tiny_patch4_window7_224")
+    verify(inputs, model, compiled_model)
```
Comment (vkovinicTT) [inline forge/test/models/pytorch/vision/swin/test_swin.py:31]: Please also run this test and if it fails put appropriate pytest mark.

**Review by vkovinicTT — CHANGES_REQUESTED** [review]

**Comment by pdeviTT** [comment]
> Friendly reminder:
> 
> * try not to load images in future, instead just create random tensor, as we don't want to have external images in our repo :)

Sure. Will try to load as random tensor in future models :))

File: forge/test/models/pytorch/vision/swin/test_swin.py (hunk 2 above)
Comment (pdeviTT) [reply forge/test/models/pytorch/vision/swin/test_swin.py:31]: will check and add failure reason in pytest mark

**Review by vkovinicTT — APPROVED** [review]

**Pull request merged**